programs/stake/src/stake_state.rs

```diff
@@ -761,11 +761,22 @@ impl<'a> StakeAccount for KeyedAccount<'a> {
             match self.state()? {
                 StakeState::Stake(meta, mut stake) => {
                     meta.authorized.check(signers, StakeAuthorize::Staker)?;
+                    let original_lamports_per_byte_year = calculate_lamports_per_byte_year(
```

Comment:
Do you anticipate this value being used otherwise?  If not, it might simplify things a bit to combine the two utility functions into one `fn calculate_split_rent_exempt_reserve(source: &KeyedAccount, split: &KeyedAccount) -> u64`

Reply:
No, I hope it won't be used otherwise. Fair point, will squash.

Reply:
Updated